### ISSUE TITLE
Can't find javascript file and css file when refresh http://localhost:9001/object-browser/bucket

### ISSUE THREAD

**Issue by sujingclg** [minio/console#967]
![捕获](https://user-images.githubusercontent.com/42365729/130329813-2c5a895d-fa04-47a4-9559-b58a5977d189.PNG)

To avoid this, “homepage”:"." should be removed from package.json file.

**Comment by harshavardhana** [comment]
> ![捕获](https://user-images.githubusercontent.com/42365729/130329813-2c5a895d-fa04-47a4-9559-b58a5977d189.PNG)
> 
> To avoid this, “homepage”:"." should be removed from package.json file.

you seem to have some proxy blocking the content.

**Comment by dvaldivia** [comment]
this seems to happen if you refresh on a given route, I'll investigate, but we do need to support `homepage: "."` to be able to load the console in a subdirectory

**Comment by dvaldivia** [comment]
one of the most straight forwad solution is if we abandon `/object-browser` style routing in favor of `/#object-browser` pound based routes, thoughts @bexsoft ?

**Comment by sujingclg** [comment]
> one of the most straight forwad solution is if we abandon `/object-browser` style routing in favor of `/#object-browser` pound based routes, thoughts @bexsoft ?

But for me, I always use front end framework like umijs to develop React apps. I think cra is really not a good choice to develop production application. In contrast, vue-cli is much better then cra I think. I just compare CLI between cra and vue-cli, not React vs Vue. :D

**Comment by Pamalosebi** [comment]
I have the same issue. But the Chrome error message sais "and strict MIME type checking is enabled".
So, the next step I tried was to remove the "x-content-type-options" header.
And the result actually changed. -->
![image](https://user-images.githubusercontent.com/49550055/131689702-629c55ac-dabc-49c5-b367-69e5752d7417.png)

The last working version of minio was "RELEASE.2021-07-30T00-02-00Z".
Everything after that is thowing the same messages.

**Comment by harshavardhana** [comment]
Yes this is a regression in console UI after adding relative URL based asset loading we are working on a fix.
